client/openocd_rpc.py

```diff
@@ -144,6 +144,9 @@ def read_word(self, address):
     def _read_word_locked(self, address):
         """Reads a 32-bit word from given address."""
         out = self._send_command_locked('ocd_mdw 0x%x' % address)
+        while out.startswith(b'DAP'):
```

Comment:
So it looks like when it fails with the /DAP.*/, the resulting output looks like: "DAP transaction stalled (WAIT) - slowing down\n0x1000000c". How do you feel about adding an optional arg to OpenOcdRpc that will take a regexp or list of regexps of lines to ignore when reading output back from the "RPC" port? And then we pass those regexps in from command line arg to buffy client?

(The righter approach would be to fix up openocd to not give us garbage as result of reading values, but that's too much yak shaving.)